### DISCUSSION TITLE
how to run honojs with fastly adapter locally using viceroy

### DISCUSSION THREAD

**Discussion by sandys** [honojs/hono#1603] [General]
https://github.com/fastly/Viceroy viceroy is fastly's tool for local development.

since honojs works on fastly as well, how to get it to work using viceroy ? i know it is a two step process (compile to wasm and then execute viceroy), but would like to do so for test harness.

**Comment by yusukebe** [comment]
Hi @sandys 

By using [the fastly starter](https://github.com/honojs/starter/tree/main/templates/fastly), you can run it on Viceroy. Did you try to run the [create-hono](https://github.com/honojs/create-hono) command?
